app/views/invoices/_form.html.haml

```diff
@@ -7,7 +7,9 @@
   = f.hidden_field :recipient_id
```

Comment:
Das ist biz doof dass wir hier das Template überschreiben.. Ich glaube der einzige Unterschied ist, dass hier das `hide_total` Feld nicht angezeigt wird. Leider ist `gem deface` in Hitobito keine Option aber man könnte das `hide_total` im Core in ein eigenes Partial schieben und dann hier leer überschreiben

Reply:
Das `hide_total` Feld wird einfach weiter unten als «Spende» angezeigt, das macht das ganze irgendwie noch unnötiger: 

![image](https://github.com/user-attachments/assets/3ee79faf-05c2-4726-9952-470fee6e2584)

Reply:
@TheWalkingLeek trotzdem im Core versuchen ein eigenes Partial zu machen?

Reply:
@ThomasEllenberger ich würde das gerne noch kurz mit dir nach dem Daily anschauen, wenns geht?

Reply:
Achso ich kann mich noch erinnern dass das dem SWW damals tatsächlich wichtig war dass die Reihenfolge der Felder so aussieht...
Wenn das mitlerweile für sie nicht mehr so ausschlaggebend ist bitte in Partials aufteilen und sonst haben wir glaub praktisch nur die Option die view zu überschreiben :/